tests/validator/validator.cpp

```diff
@@ -1522,7 +1466,7 @@ TEST(Validator, validMathCnElementsMissingCellMLNamespace)
         "  </apply>\n"
         "</math>\n";
 
-    const std::vector<std::string> expectedErrors {
+    const std::vector<std::string> expectedErrors = {
```

Comment:
is there a convention around initialising through the constructor cf through assignment?  in an earlier PR the equals were all removed ...?

Reply:
I don't remember which PR that was.  I noticed there was a difference in how they we declared and made them all the same, went with the majority format.